man/systemd.network.xml

```diff
@@ -1899,6 +1899,14 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RetryUplink=</varname></term>
+
+        <listitem><para>Takes a boolean. When true, DHCPv4 server dynamically reconfigures DNS, SIP and NTP information
+        when uplink is available by sending a forcerenew message to DHCPv4 clients. Defaults to <literal>no</literal>.
+        </para></listitem>
+      </varlistentry>
```

Comment:
hmm, shouldn't this be the behaviour we implement by unconditionally?

or if it is conditional, then at least make it opt-out rather than opt-in? It sounds a lot more sensible to me to keep propagating current data instead of sticking to out-of-date data.